ProductReleaseAnnouncement/product_rel_announcement.py

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+# vi: set ft=python :
+"""
+Auto-generate social media posts for Medik8s release announcements by filling in the latest tags and
+release information.
+
+Usage:
+    product_rel_announcement.py --markdown
+    product_rel_announcement.py --html
+    product_rel_announcement.py --html --markdown
+
+Options:
+    -m, --markdown  Write the release announcement file release.md, in Markdown format
+    -w, --html      Write the release announcement file release.html, in HTML format
+"""
+import requests
+import markdown
+from docopt import docopt
+
+
+def main():
+    arguments = docopt(__doc__)
+    NMO = get_latest_version("node-maintenance-operator")
+    NHC = get_latest_version("node-healthcheck-operator")
+    SNR = get_latest_version("self-node-remediation")
+    FAR = get_latest_version("fence-agents-remediation")
+    MDR = get_latest_version("machine-deletion-remediation")
+
+    TEMPLATE = f"""
+Medik8s New Releases
+
+The Medik8s team is thrilled to announce the new releases of ours operators
```

Comment:
NIT: Colon (`:`) at the end of the sentence for listing the operators releases.

```suggestion
The Medik8s team is thrilled to announce the new releases of ours operators:
```